forc-plugins/forc-doc/Cargo.toml

```diff
@@ -27,6 +27,7 @@ sway-features.workspace = true
 sway-lsp.workspace = true
 sway-types.workspace = true
 swayfmt.workspace = true
+tokio = { workspace = true, features = ["macros", "rt-multi-thread"] }
```

Comment:
seems weird you'd need to include a full async runtime just to deal with some http errors, surely there are some synchronous facilities to do that even if the given library is written with an async client in mind.

Can you specifically explain the reasoning here?

Reply:
There was a bug in fork-pkg which I resolved here https://github.com/FuelLabs/sway/pull/7246 so this change shouldn't need to happen anymore.